blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How a hack week project made Sourcegraph.com easier to use, by searching the top 10k open source repositories on GitHub by default."
+author: Issac Trotts, Software Engineer at Sourcegraph
+publishDate: 2019-08-16T00:00-11:55
+tags: [
+  blog
+]
+slug: Search-the-top-10000-open-source-repositories-Sourcegraph-com
+heroImage: /blog/sourcegraph-com-usage.png
+published: true
+---
+
+<p style="text-align: center">
+  <img src="/blog/sourcegraph-com-usage.png" alt="My top feature request is: no new features. Just fix what's broken">
+  <br/><small>Sourcegraph.com stats as of August 2019</small>
+</p>
+
+Wouldn't it be awesome if you could review, browse, and search code from the top ten thousand open source repositories on GitHub? You can, using [Sourcegraph.com](https://sourcegraph.com), and we've now made it easier to use!
+
+## Using Sourcegraph.com like a regular search engine
+
+[Sourcegraph.com](https://sourcegraph.com) indexes over 4 million open source repositories on GitHub! The only problem, was that a [`repo` filter](https://docs.sourcegraph.com/user/search/queries) was required to narrow the repository set to a manageable number, the downside of which, meant you had to know what repositories to search. This meant [Sourcegraph.com](https://sourcegraph.com) was missing out on a key use case⁠—random usage discovery.
+
+Let's say you've recently heard about Google's [Distroless container image](https://github.com/GoogleContainerTools/distroless), and are curious about its usage. Previously, without specifying any repsoitories, your search would've returned this:
```

Comment:
@ryan-blunden I think we should have an automated spell checker like https://github.com/liamzdenek/go-spellcheck

```suggestion
Let's say you've recently heard about Google's [Distroless container image](https://github.com/GoogleContainerTools/distroless), and are curious about its usage. Previously, without specifying any repositories, your search would've returned this:
```

Reply:
Added https://github.com/sourcegraph/about/issues/221